js/colourpicker.js

```diff
@@ -668,160 +670,163 @@
+    // gets a flipped version of colsToNames so that we can search by colors for names
```

Comment:
the description of the function should be describing the function, not the use for it that comes after, does that make sense?